README.md

```diff
@@ -69,3 +68,13 @@ Currently the backend consists of an auth endpoints and endpoints for communicat
 ## CI
 
 Pipelines will run when pull requests are created or commits are pushed to the origin.
+
+## Deployment
+
+The frontend and backend applications are deployed to two Azure Web Applications. The frontend can be accessed [here](https://playman.azurewebsites.net/) These can be updated locally by:
+
+1. Building new Docker images with (from the relevant frontend/backend directory) `docker build --target production --tag calpin/playlist-manager-"frontend/backend":prod .`
+2. Pushing the new image to Docker Hub `docker push calpin/playlist-manager-"frontend/backend":prod`
+3. Sending a curl request to the appropriate Webhook url:
+    - Frontend: `curl -dH -X POST "https://\$PlayMan:d17p8pbD0t6xpCdep4tPdetcbsTlZiomZ6kwAwuJB9XaXwjE5hmmgSpBTYqQ@playman.scm.azurewebsites.net/api/registry/webhook"`
+    - Backend: `curl -dH -X POST "https://\$PlayManBackend:aBPc2NHMrYLEHQtmPB1JhQKgtsonxezPjliwxptyx2142STiv91aRxmnmKe6@playmanbackend.scm.azurewebsites.net/api/registry/webhook"`
```

Comment:
These webhooks contain sensitive values! Before merging, make sure to rotate these values in the Azure portal now, under:
* App service -> Deployment Center -> FTPS Credentials -> Password
And then update these instructions:
```suggestion
    - Frontend: `curl -dH -X POST "https://\$PlayMan:<password>@playman.scm.azurewebsites.net/api/registry/webhook"`
    - Backend: `curl -dH -X POST "https://\$PlayManBackend:<password>@playmanbackend.scm.azurewebsites.net/api/registry/webhook"`
```

Reply:
I actually spotted this during the next project exercise, so this is updated in the following PR.
While I should have probably spotted this myself, there's a warning in the next exercise about the Webhook Url containing sensitive values which prompted me to remove this, but it might be worth adding a note in [this section](https://project-exercises.devops.corndel.com/exercises/m8_exercise#:~:text=be%20going%20wrong-,Part%203%3A%20Update%20the%20container,-When%20creating%20an) that these webhooks shouldn't be committed to source control.